### PR TITLE
BD-1528: Document campaign search syntax

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

#### Description of Change:
Document campaign search syntax, specifically searching by API identifier

1. Adds a **Searching for campaigns** page to the **Managing Campaigns** section

2. Adds an entry to the Campaigns **FAQ** page

3. Adds a use case to the **API Identifier Types** page

Closes #**BD-1528**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

### CODE REVIEW

File: _docs/_api/identifier_types.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ You can find your campaign ID one of two ways:
 - Grab high-level aggregate stats on campaign performance
 - Grab details on a specific campaign
 - With Currents to bring in user-level data for a "bigger picture" approach to campaigns
-- With API trigger delivery in order to collect statistics for transactional messages
+- With API-triggered delivery in order to collect statistics for transactional messages
+- To [search for a specific campaign]({{site.baseurl}}/user_guide/engagement_tools/campaigns/managing_campaigns/search_campaigns/#search-syntax) on the **Campaigns** page using the syntax `api_id:YOUR_API_ID`
```
Comment (bre-fitzgerald) [inline _docs/_api/identifier_types.md:78]:
```suggestion
- To [search for a specific campaign]({{site.baseurl}}/user_guide/engagement_tools/campaigns/managing_campaigns/search_campaigns/#search-syntax) on the **Campaigns** page using the filter `api_id:YOUR_API_ID`
```

File: _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/search_campaigns.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Filters
+
+Use the filters in the side menu to group results by creator, editor, send dates, or channel, or select **Only Show Mine** to limit the results of your search to campaigns you've created. You can also filter by status and [tags]({{site.baseurl}}/user_guide/administrative/app_settings/manage_app_group/tags/) to further narrow down your results.
+
+![][2]
+
+Expand the search dropdown to filter by last editor, target segment, messaging channel, or date.
+
+![][3]
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/search_campaigns.md:30]:
```suggestion
![][3]
```
Minor nit here, but could we crop the image a bit to remove the small white/gray space around the border?

File: _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/search_campaigns.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Search syntax
+
+Selecting a campaign filter will automatically add the appropriate syntax to the search field, however you can manually enter in these filters as well. When using manual search, the syntax is the filter name, followed by a semicolon, followed by your input. For example, to search for push campaigns, enter `channel:push`.
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/search_campaigns.md:34]:
```suggestion
Selecting a campaign filter will automatically add the appropriate syntax to the search field. However, you can manually enter these filters as well. When using manual search, the syntax is the filter name, followed by a semicolon, followed by your input. For example, to search for push campaigns, enter `channel:push`.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks great! left a couple suggestions for your review! 👍

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**